### ISSUE TITLE
Add present-tense aliases to support callable Actors.

### ISSUE THREAD

**Issue by perrygoy** [ScreenPyHQ/screenpy_selenium#26]
Actors can now be called directly with their performances, which means present-tense aliases are needed! For example:
```python
the_actor(
    Enters.the_text(username).into_the(USERNAME_FIELD),
    Enters.the_secret(password).into_the(PASSWORD_FIELD),
    Clicks.on_the(LOGIN_BUTTON), 
    ...,
)
```

**Comment by perrygoy** [comment]
P.S. thanks for the catch @pjbarbatsis!

**Comment by perrygoy** [comment]
Closed with #27!
